opengrok-indexer/src/main/java/org/opengrok/indexer/analysis/sql/PLSQLAnalyzerFactory.java

```diff
@@ -36,7 +36,8 @@
         "PLD",
         "PLB",
         "PKB",
-        "PCK"
+        "PCK",
+        "PKG"
```

Comment:
PKG is too generic (and mostly used for packages, not for sql) ... I wouldn't do this, from my side this can go to your own config per repo, but not for all users.
So please remove this, most users would be only bugged by this.